systemtests/scripts/cleanup

```diff
@@ -24,9 +24,7 @@ rm -rf /tmp/chrome-user-data-*
 
 "${rscripts}/bareos" stop >/dev/null
 
-${scripts}/drop_bareos_tables ${DBTYPE} >/dev/null
-${scripts}/drop_bareos_database ${DBTYPE} >/dev/null
-echo "creating database (${DBTYPE})"
-${scripts}/create_bareos_database ${DBTYPE} >/dev/null
-${scripts}/make_bareos_tables ${DBTYPE} >/dev/null
-${scripts}/grant_bareos_privileges ${DBTYPE} >/dev/null
+${scripts}/drop_bareos_tables ${DBTYPE} >/dev/null 2>&1
```

Comment:
cleanup is called multiple times and pollutes the output. I therefore disabled the output and have not find any problems with this. In case, problems do arise, output can be enabled again (maybe in a from, that only real errors are shown).